regression/cbmc-java/NondetInitValidate/Test.java

```diff
@@ -0,0 +1,31 @@
+import org.cprover.CProver;
+
+class Test {
+    int size;
+    int[] data1;
+    int[] data2;
+
+    void cproverValidate() {
+        // This specifies invariants about object of this class.
+        // This avoids finding spurious bugs.
+        CProver.assume(data1 != null && data2 != null && size == data1.length + data2.length);
+    }
```

Comment:
This test only assumes something about the non-det initialization.

I would add another test for a common case: actually performing the non-det initialization in this method. Talk to @antlechner on how to initialize the keys of a hash table.

I in fact then support that you actually rename this method to `cproverNondetInit()`

Reply:
We are still not sure about what name to choose. The use case I had in mind was to only add assumptions about the data, so `cproverValidate` would make sense. It is actually called after the `gen_nondet_init` things are done, so `cproverNondetInit` could be confusing. I will check with @antlechner what would be needed for hash tables.